app/javascript/src/Navigation/components/VerticalNav.jsx

```diff
@@ -20,19 +22,30 @@ type Section = Item & {
 type Props = {
   sections: Section[],
   activeSection: ?string,
-  activeItem: ?string
+  activeItem: ?string,
+  currentApi: ?Api
 }
 
-const VerticalNav = ({ sections, activeSection, activeItem }: Props) => (
+const VerticalNav = ({ sections, activeSection, activeItem, currentApi }: Props) => (
   <div className="pf-c-page__sidebar-body">
     <Nav id='mainmenu' theme='dark'>
-      <NavList>
-        {sections.map(({ id, title, path, items, outOfDateConfig }) => {
-          return items
-            ? <NavSection title={title} isSectionActive={id === activeSection} activeItem={activeItem} items={items} key={title} outOfDateConfig={outOfDateConfig}/>
-            : <NavItem to={path} isActive={activeSection === id} key={title}>{title}</NavItem>
-        })}
-      </NavList>
+      { currentApi ? (
+        <NavGroup title={currentApi.name}>
+          {sections.map(({ id, title, path, items, outOfDateConfig }) => {
+            return items
+              ? <NavSection title={title} isSectionActive={id === activeSection} activeItem={activeItem} items={items} key={title} outOfDateConfig={outOfDateConfig}/>
+              : <NavItem to={path} isActive={activeSection === id} key={title}>{title}</NavItem>
+          })}
```

Comment:
Good! But it is yelling _DRY!_ 😀 Let's extract the "children" part and leave it like:
```jsx
{currentApi ? <NavGroup title={currentApi.name}>{sections}</NavGroup> : <NavList>{sections}</NavList>}
```

We could even encapsulate this logic in its own `CustomNavGroup`.

Reply:
updated!